### DISCUSSION TITLE
Grove - PIR Motion Sensor | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#760] [Q&A]
# Grove - PIR Motion Sensor | Seeed Studio Wiki

Grove - PIR Motion Sensor

https://wiki.seeedstudio.com/Grove-PIR_Motion_Sensor/

**Comment by KazuhiroJinnouchi** [comment]
In the following part of the explanation of FAQs A1, R2 and R6 are reversed in the explanation of [PDF] Grove - PIR Motion Sensor Eagle V1.2 PCB.
Which is correct?
>　R2: used to adjust the detecting distance(the AMP coefficient, 2MΩ). R6: used to adjust the holding time(the trigger duty, 100KΩ).
